modules/nw-dns-forward.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-forward_{context}"]
+= Using DNS forwarding on user-provisioned infrastructure in AWS
+
+You can use forwarding with OpenShift DNS to resolve name queries for your other internal devices using the DNS servers in your data center.
```

Comment:
By default, the cluster DNS service will resolve internal cluster names and forward requests for external names to the name servers that are specified in `/etc/resolv.conf`.  The purpose of the DNS forwarding API is to allow overriding the forwarding configuration on a per-zone basis, by specifying which name server should be used for a given zone.

Reply:
Would it be appropriate to introduce DNS forwarding as a condensed version of your explanation, .e.g.,:

"You can use DNS forwarding to override the forwarding configuration identified in `etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone."

Or would you mind if I use your explanation?

Reply:
I think that looks better, yeah.